src/vm/instruction.test.ts

```diff
@@ -135,6 +189,29 @@ describe('Instruction', () => {
       expect(instruction).toEqual(expected);
     });
 
+    test('should correctly decode the cairo instruction CALL from LC example', () => {
```

Comment:
```suggestion
    test('should correctly decode the cairo instruction CALL - ported from lambdaclass/cairo-vm_in_go', () => {
```

Reply:
that way we can reuse this norm to say
ported from lambdaclass/cairo-vm & nethermind/cairovmgo